Basic-Car-Maintenance.xcodeproj/project.pbxproj

```diff
@@ -99,13 +100,17 @@
 
 /* Begin PBXFileReference section */
 		023057F12ACFAD79006C5A73 /* EditEventDetailView.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = EditEventDetailView.swift; sourceTree = "<group>"; };
+<<<<<<< HEAD
+		8288B89A2AE5C5C900AA21F9 /* AddVehicleViewUITest.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = AddVehicleViewUITest.swift; sourceTree = "<group>"; };
+=======
```

Comment:
This is what a merge conflict looks like, this is something you have to fix, and not commit